README.md

```diff
@@ -48,7 +48,7 @@ Pull the update into your local master branch:
 git pull origin master
 ```
 
-Push the update live to gh-pages:
+Push the update live to the GH-pages:
```

Comment:
gh-pages is the name of the branch  and should remain lower case, if anything put ` around it to highlight it as code